docker-dev.txt

```diff
@@ -31,11 +31,8 @@ You now have semesterly running. If this is the first time, you will want some d
 
 Getting JHU data for a given term
 * In a new terminal run the following
-docker exec -it $(docker ps -q -f ancestor=semesterly) /bin/bash
```

Comment:
Looks like the version we want is the Noah's copy, and this PR is overwriting. We should sync this file to Jhu repo